packages/calcite-components/src/utils/openCloseComponent.ts

```diff
@@ -14,6 +14,11 @@ export interface OpenCloseComponent {
    */
   open?: boolean;
 
+  /**
+   * When true, the component is open.
+   */
+  opened?: boolean;
```

Comment:
@Elijbet Let's look into refactoring this to follow `<propName>Prop`, similar to other props, so components can specify their source-of-truth open prop. #proppropprop